### ISSUE TITLE
MultipleQuickslotBars quickslot>quickbar priority order change

### ISSUE THREAD

**Issue by ShadoweCZ** [sinai-dev/Outward-Mods#21]
if u set up quickbar switch key and skill keybind to the same key then the quickbar is changed first befor the quickslot is used ... 

i wanted to make it so with 1 click i can switch weapon (that is in quickslot) and that will also switch quickbar to next one with skills for that weapon (example if i have skill 5 set as weapon switch to bow and 5 also as quickbar switch so when i equip the bow i get new quickbar full of bow skills)
 it dosnt work this way tho, ... what it does is it switches the bars first and then uses skill 5 from the other bar which kinda dosnt make sense because you dont see what do you have in the next bar, especialy if u use more of them

TLDR could u make it so skill/item quickslot is used first before quickbar change ?
